dbt_meshify/storage/yaml_editors.py

```diff
@@ -248,14 +251,18 @@ def add_model_version(
         # read the yml file
         # pass empty dict if no file contents returned
         models_yml = self.file_manager.read_file(yml_path) or {}
-        updated_yml = self.add_model_version_to_yml(
-            model_name=self.model_node.name,
-            models_yml=models_yml,
-            prerelease=prerelease,
-            defined_in=defined_in,
-        )
-        # write the updated yml to the file
-        self.file_manager.write_file(yml_path, updated_yml)
+        try:
+            updated_yml = self.add_model_version_to_yml(
+                model_name=self.model_node.name,
+                models_yml=models_yml,
+                prerelease=prerelease,
+                defined_in=defined_in,
+            )
+            # write the updated yml to the file
+            self.file_manager.write_file(yml_path, updated_yml)
+            logger.info("Model version added to model yml")
+        except Exception as e:
+            logger.error(f"Error adding model version to model yml: {e}")
```

Comment:
Currently, this try/except eats the exception. Am I correct in believe that we want to re-raise the exception after logging?